zeebe/auth/src/test/java/io/camunda/zeebe/auth/JwtAuthorizationTest.java

```diff
@@ -110,6 +110,7 @@ public void shouldFailJwtTokenValidationWithNoAuthorizedTenants() {
             .withIssuer(DEFAULT_ISSUER)
             .withAudience(DEFAULT_AUDIENCE)
             .withSubject(DEFAULT_SUBJECT)
+            .withClaim(Authorization.AUTHORIZED_USER_KEY, 1234L)
```

Comment:
🔧 I'm a bit surprised to see this in this test case. According to the name this verifies `shouldFailJwtTokenValidationWithNoAuthorizedTenants`.

It's more clear to add a new test case to try adding the authorized user key.

Reply:
This was accidentally left in for this test, I need to add a separate one for the user key as you suggest :)